src/payment/data/reducers.js

```diff
@@ -57,8 +60,39 @@ const basket = (state = basketInitialState, action = null) => {
   return state;
 };
 
+const captureContextInitialState = {
+  loading: true,
+  loaded: false,
+  isCaptureKeyProcessing: false,
+  products: [],
+};
+
+const captureKey = (state = captureContextInitialState, action = null) => {
+  if (action !== null) {
+    switch (action.type) {
+      case fetchCaptureKey.TRIGGER: return { ...state, loading: true };
```

Comment:
Should this set `loaded: false` so that it's consistent when/if we need to reload during the timeout? Or, alternately, should there just be a single flag (`loaded`) which means `loading` when it's false, and `loaded` when it's true?

Reply:
Is `loading` used anywhere? Or are you just setting up for future work?

Reply:
I ended up not needing the `loading` bits and dropped them.